graviti/schema/builtin.py

```diff
@@ -32,7 +28,7 @@
 }
 
 
-class PortexBuiltinType(PortexType):
+class PortexBuiltinType(PortexType):  # pylint: disable=abstract-method
```

Comment:
where the pylint warning comes from?

Reply:
Because this class does not implement the `to_pyarrow` method which is not implemented in its parent class.